test/system/505-networking-pasta.bats

```diff
@@ -448,6 +448,10 @@ function teardown() {
 }
 
 @test "podman networking with pasta(1) - Local forwarder, IPv4" {
+    if [[ "$CIRRUS_CI" == "true" ]] && [[ "$DISTRO_NV" == "debian-13"]]; then
```

Comment:
This is bash, not a programming language :-(. You need a space between `"debian-13"` and `]]`.

Reply:
Oof, I hate it when I do that.  Thanks.